milestone1.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
```

Comment:
We want our home page to be named index.html, this is the standard name for the main page of a website in html